test/spec/search.js

```diff
@@ -28,7 +28,45 @@ test('Search should call the algolia client according to the number of refinemen
       JSON.parse(JSON.stringify(testData.responseHelper)),
       'should be equal'
     );
+
+    var cityValues = data.getFacetValues('city');
+    var expectedCityValues = [
+      {name: 'Paris', count: 3, isRefined: true},
+      {name: 'New York', count: 1, isRefined: true},
+      {name: 'San Francisco', count: 1, isRefined: false}
+    ];
+
+    t.deepEqual(
+      cityValues,
+      expectedCityValues,
+      'Facet values for "city" should be correctly ordered using the default sort');
+
+    var cityValuesCustom = data.getFacetValues('city', {sortBy: ['name:desc']});
```

Comment:
Could we also add a test sorting on `['count:asc', 'name:asc']` to ensure the magical lodash function is working?